### ISSUE TITLE
Update question: "Does a page need to validate HTML?" with accurate information

### ISSUE THREAD

**Issue by MaximPerry** [cenw-wscoe/WCAG-in-Plain-Language#28]
Not every HTML error are necessarily WCAG 4.1.1 - Parsing issues. It should be indicated so.
